README.md

```diff
@@ -162,23 +162,23 @@ need to be provided in the 'Authorization' section of all request headers.
 
 Authorization header format:
 
-Key:    Authorization
-Value:  Token 3f26b15ee5c4723ecd91ddde5809a248c1f1a5b5
+Key: Authorization
+Value: Token 3f26b15ee5c4723ecd91ddde5809a248c1f1a5b5
 
 ### Testing
 
 #### Creating unit tests
 
-* Ensure test files are created in a relevant sub-directory under the test directory of the app under test.
-* Ensure all test files are prefixed with "test_" and match the name of the module/class that this test file tests.
-* Ensure that import unittest is brought into the test file.
-* Base your test class on unittest.TestCase.
-* Add the new test file to the __init__.py file so that the unit test runner picks the new tests up.
+- Ensure test files are created in a relevant sub-directory under the test directory of the app under test.
+- Ensure all test files are prefixed with "test\_" and match the name of the module/class that this test file tests.
+- Ensure that import unittest is brought into the test file.
+- Base your test class on unittest.TestCase.
+- Add the new test file to the **init**.py file so that the unit test runner picks the new tests up.
```

Comment:
Think there might a mistake here
The file name is **__init__.py** because it a special method in python

Reply:
Well spotted Matt. I think this is autoformatting. We should be wrapping the file name in backticks e.g. `__init__.py`. The whole `README.md` file needs a bit of moderation.